Adapter/Translator.php

```diff
@@ -105,7 +106,7 @@ public function trans($id, array $parameters = array(), $domain = null, $locale
      * @param string      $id         The message id (may also be an object that can be cast to string)
```

Comment:
+:
You cannot use variables for $id and $domain.
eg: $message = 'hello'; $domain = 'AdminTab'; transchoice($message, 2, [], $domain); IS FALSE
prefer: transchoice('hello', 2, [], 'AdminTab'); TO ENSURE TRANS PARSER WILL FIND YOUR MESSAGE.